plugins/woocommerce/client/legacy/js/admin/meta-boxes-product.js

```diff
@@ -1069,6 +1069,57 @@ jQuery( function ( $ ) {
 		return false;
 	} );
 
+	// Add gpt integration button
+	$( '#wp-content-media-buttons' ).append(
```

Comment:
Isn't there a way to add this in php? Looks very hacky here.

Reply:
Good call! I had to add this [hacky DOM management](https://github.com/woocommerce/woocommerce/pull/37716/files#diff-6cb6042af2aa3f5dd0e577f64c0e8f7364b5e0571087f6e65de6022c082f36ddR667-R677) to move the buttons, but it was only that 😄